.jenkins/Jenkinsfile-pr

```diff
@@ -124,6 +127,39 @@ pipeline {
             }
 
         }
+        stage('workaround olm-manifest step 1') {
```

Comment:
I wonder if a more uniform step would be to edit the Makefiles to Allow setting an INTERNAL_DOCKER_REGISTRY (which by default == DOCKER_REGISTRY) and override that for the olm-manifest image. Then these steps should be unnecessary and we'll save issues in other DSL/pipelines as well. I can share a pr of what i mean.

Reply:
hmm, that PR is welcome @lulf :) , note that `make docker_build` and `make docker_tag` have to be run with different DOCKER_REGISTRY values in order to push to external registry but having CSV pointing to internal registry

Reply:
I tried to modify the makefiles, but there wasn't any clean/easy way to do that. However, I recommend instead to

* Build with internal registry
* Run `make DOCKER_REGISTRY=${PUBLIC_REGISTRY} templates docker_tag docker_push`